shared/macros-bash.jinja

```diff
@@ -1649,39 +1649,31 @@ fi
     Remediation for grub2 bootloader arguments
 #}}
 {{% macro grub2_bootloader_argument_remediation(ARG_NAME, ARG_NAME_VALUE) %}}
-{{% if product in ["rhel7", "ol7", "rhel9"] or 'ubuntu' in product %}}
-{{% if '/' in ARG_NAME %}}
-{{{ raise("ARG_NAME (" + ARG_NAME + ") uses sed path separator (/) in " + rule_id) }}}
-{{% elif '/' in ARG_NAME_VALUE %}}
-{{{ raise("ARG_NAME_VALUE (" + ARG_NAME_VALUE + ") uses sed path separator (/) in " + rule_id) }}}
-{{% endif %}}
+{{% set grub_helper_executable = "grubby" -%}}
+{{% set grub_helper_args = ["--update-kernel=ALL", "--args=" ~ ARG_NAME_VALUE] -%}}
+
+{{%- macro update_etc_default_grub_manually() -%}}
```

Comment:
Macro inside a macro can confuse some people, especially in Jinja which doesn't guide people visually by indentation. I'd prefer to avoid macros inside macros.

Reply:
I will extract the macro out